javaHapiValidatorLambda/src/main/java/software/amazon/fwoa/Validator.java

```diff
@@ -64,22 +61,24 @@
     private final FhirContext ctx;
 
     private final String fhirVersion;
-    private final String implementationGuidesFolder;
+    private final List<IGObject> implementationGuidesIndices;
+    private final List<IGObject> implementationGuidesResources;
 
     public Validator() {
-        this(FHIR_R4, DEFAULT_IMPLEMENTATION_GUIDES_FOLDER);
+        this(FHIR_R4, DEFAULT_IMPLEMENTATION_GUIDES, DEFAULT_IMPLEMENTATION_GUIDES);
     }
 
     public Validator(String fhirVersion) {
-        this(fhirVersion, DEFAULT_IMPLEMENTATION_GUIDES_FOLDER);
+        this(fhirVersion, DEFAULT_IMPLEMENTATION_GUIDES, DEFAULT_IMPLEMENTATION_GUIDES);
     }
 
-    public Validator(String fhirVersion, String implementationGuidesFolder) {
```

Comment:
Do we need a constructor `Validator(List<IGObject> indices, List<IGObject> definitions)` as well?